pkg/git/dummy/download.go

```diff
@@ -41,5 +40,11 @@ func (f FileService) EncodeURL(org, repo, path string) string {
 
 // DecodeURL decodes a URL
 func (f FileService) DecodeURL(url string) (org, repo, path string) {
-	return (&github.FileService{}).DecodeURL(url)
+	targetExpression := ".*/repos/(.+)/(.+)/contents/(.+)"
```

Comment:
Why not just combine these two lines?

Reply:
a) I lifted the block from github where it was using fmt.Sprintf to build the regex.
b) I think it's a little more legible to see the expression outside the actual regexp call.

But mostly a).  ;)